asciidoc/courses/llm-vectors-unstructured/modules/3-unstructured-data/lessons/2-chunking/questions/1-chunksize.adoc

```diff
@@ -0,0 +1,21 @@
+[.question]
+= Chunk size
+
+True or False - embedding models can handle any length of text data, so chunking is not always necessary.
+
+* [ ] True
+* [x] False
+
+
+[TIP,role=hint]
+.Hint
+====
+Smaller pieces of data are easier to work with and process.
+====
+
+
+[TIP,role=solution]
+.Solution
+====
+The answer is `False`. Embedding models have size (token) limits and can only handle a certain amount of data. 
+====
```

Comment:
That's three true or false questions in a row.  Could you mix it up a bit, please?

Reply:
Yes, replaced one of the questions